### DISCUSSION TITLE
Stylelint Deprecated Config

### DISCUSSION THREAD

**Discussion by juliankasimir** [eviden-parallel/community#15] [General]
[.stylelintrc](https://github.com/eviden-parallel/stylelint-config/blob/main/.stylelintrc) contains a lot of rules that are deprecated for v15, e.g.:

https://stylelint.io/user-guide/rules/at-rule-name-case/

**Comment by raj19joshi** [comment]
Agree, we are fixing it in STC App, and same can be removed from Parallel after discussion, as in next stylelint major release they will remove the rules from Stylelint. It quotes:
When we created these rules, pretty printers (like [Prettier](https://prettier.io/)) didn't exist. They now offer a better way to consistently format code, especially whitespace. Linters and pretty printers are complementary tools that work together to help you write consistent and error-free code.

[https://stylelint.io/migration-guide/to-15/](https://stylelint.io/migration-guide/to-15/)

**Comment by raj19joshi** [comment]
Erledigt.

**Comment by Th3S4mur41** [comment]
Fixed in https://github.com/eviden-parallel/stylelint-config/pull/169
